docs/source/hyper-parameter-search.rst

```diff
@@ -185,34 +238,33 @@ expensive early steps, this can be a big win when performing a grid search.
 Incremental Hyperparameter Optimization
 ---------------------------------------
 
-The second category of hyperparameter optimization uses *incremental*
-hyperparameter optimization. These should be used when your full dataset doesn't
-fit in memory on a single machine.
-
 .. autosummary::
    dask_ml.model_selection.IncrementalSearchCV
 
-Broadly speaking, incremental optimization starts with a batch of models (underlying
-estimators and hyperparameter combinations) and repeatedly calls the underlying estimator's
-``partial_fit`` method with batches of data.
-
 .. note::
 
    These estimators require the optional ``distributed`` library.
 
-Here's an example training on a "large" dataset (a Dask array) with the
-``IncrementalSearchCV``.
+These are make repeated calls to the ``partial_fit`` method of the estimator.
+Naturally, these classes determine when to stop calling ``partial_fit`` by
+`adapting to previous calls`. The most basic level of this is to stop training
+if the score doens't improve, which ``IncrementalSearchCV`` does.
+
+
+Basic use
+^^^^^^^^^
 
 .. ipython:: python
 
     from dask.distributed import Client
     client = Client()
     import numpy as np
     from dask_ml.datasets import make_classification
-    X, y = make_classification(n_samples=5000000, n_features=20,
-                               chunks=100000, random_state=0)
+    # X, y = make_classification(n_samples=5000000, n_features=20,
+    #                           chunks=100000, random_state=0)
+    X, y = make_classification(chunks=20, random_state=0)
 
-Our underlying estimator is an SGDClassifier. We specify a few parameters
+Our underlying estimator is an ``SGDClassifier``. We specify a few parameters
```

Comment:
Maybe make this a reference to the scikit-learn class :class:\`sklearn.linear_model.SGDClasifier\`